src/app/screens/ledger/confirmLedgerTransaction/index.tsx

```diff
@@ -11,10 +11,11 @@ import {
   signLedgerStxTransaction,
   signLedgerMixedBtcTransaction,
   satsToBtc,
+  microstacksToStx,
 } from '@secretkeylabs/xverse-core';
 import BigNumber from 'bignumber.js';
 import useWalletSelector from '@hooks/useWalletSelector';
-import { Recipient } from '@secretkeylabs/xverse-core/transactions/btc';
+import { Recipient, StacksRecipient } from '@secretkeylabs/xverse-core/transactions/btc';
```

Comment:
@dhriaznov can you point me to which PR has this `StacksRecipient` change in core. I couldn't find it in the core PR https://github.com/secretkeylabs/xverse-core/pull/212

My suggestion here is to add the type in the `transaction/stx.ts` file in core since it makes more sense for the stacks type to be there

Reply:
It looks like I didn't push the `StacksRecipient` type adding to the PR in core.

Now I moved it to `transaction/stx.ts` and pushed that code https://github.com/secretkeylabs/xverse-core/pull/212/files#diff-29fb3ccb8a9a4780fbc6835662d861640d37b464210a2c279975ff2e3f8983b0R56.